doc/modules/multiclass.rst

```diff
@@ -240,54 +112,86 @@ if you're using one of these, unless you want custom multiclass behavior:
   - :class:`sklearn.ensemble.RandomForestClassifier`
 
+.. _multiclass_classification:
+
+Multiclass classification
+=========================
 
 .. warning::
+    All classifiers in scikit-learn do multiclass classification
+    out-of-the-box. You don't need to use the :mod:`sklearn.multiclass` module
+    unless you want to experiment with different multiclass strategies.
 
-    At present, no metric in :mod:`sklearn.metrics`
-    supports the multioutput-multiclass classification task.
-
-Multilabel classification format
-================================
-
-In multilabel learning, the joint set of binary classification tasks is
-expressed with label binary indicator array: each sample is one row of a 2d
-array of shape (n_samples, n_classes) with binary values: the one, i.e. the non
-zero elements, corresponds to the subset of labels. An array such as
-``np.array([[1, 0, 0], [0, 1, 1], [0, 0, 0]])`` represents label 0 in the first
-sample, labels 1 and 2 in the second sample, and no labels in the third sample.
-
-Producing multilabel data as a list of sets of labels may be more intuitive.
-The :class:`MultiLabelBinarizer <sklearn.preprocessing.MultiLabelBinarizer>`
-transformer can be used to convert between a collection of collections of
-labels and the indicator format.
-
-  >>> from sklearn.preprocessing import MultiLabelBinarizer
-  >>> y = [[2, 3, 4], [2], [0, 1, 3], [0, 1, 2, 3, 4], [0, 1, 2]]
-  >>> MultiLabelBinarizer().fit_transform(y)
-  array([[0, 0, 1, 1, 1],
-         [0, 0, 1, 0, 0],
-         [1, 1, 0, 1, 0],
-         [1, 1, 1, 1, 1],
-         [1, 1, 1, 0, 0]])
+**Multiclass classification** is a classification task with more than two
+classes. Each sample can only be labeled as one class.
+
+For example, classification using features extracted from a set of images of
+fruit, where each image may either be of an orange, an apple, or a pear.
+Each image is one sample and is labeled as one of the 3 possible classes.
+Multiclass classification makes the assumption that each sample is assigned
+to one and only one label - one sample cannot, for example, be both a pear
+and an apple.
+
+While all scikit-learn classifiers are capable of multiclass classification,
+the meta-estimators offered by :mod:`sklearn.multiclass`
+permit changing the way they handle more than two classes
+because this may have an effect on classifier performance
+(either in terms of generalization error or required computational resources).
+
+Target format
+-------------
+
+Valid :term:`multiclass` representations for
+:func:`~sklearn.utils.multiclass.type_of_target` (`y`) are:
+
+  - 1d or column vector containing more than two discrete values. An
+    example of a vector ``y`` for 4 samples:
+
+      >>> import numpy as np
+      >>> y = np.array(['apple', 'pear', 'apple', 'orange'])
+      >>> print(y)
+      ['apple' 'pear' 'apple' 'orange']
+
+  - Dense or sparse :term:`binary` matrix of shape ``(n_samples, n_classes)``
```

Comment:
I don't think this is a multiclass representation. I think this is a treatment of a multiclass problem as multiple binary problems. I.e. it is a recoding of the multiclass representation.

Reply:
Ah, right! Yes. This was written when I still had the misconception.

You cleared this up for me in #18145, but I didn't update this section afterwards, so thank you for pointing this out.

Reply:
I'm having trouble balancing being technically thorough with my language, yet still making things accessible to someone who may be new to these concepts. Which details to include, and which to leave out... hmm...

How about still mentioning this representation, but directing users to the `preprocessing_targets.rst` documentation instead?

```
.. warning::

    Certain estimators support binary data only. To get around this limitation,
    multiclass data can instead be represented using a :term:`label indicator matrix` 
    (which is typically used for multilabel classification). To transform multiclass data
    into this form, use :class:`~sklearn.preprocessing.LabelBinarizer`. For more 
    information, please refer to :ref:`preprocessing_targets`.
```